styles/globals.scss

```diff
@@ -4,10 +4,12 @@
 
 $nhsFont: “Frutiger W01”, Arial, sans-serif;
 $nhsBlue: #005eb8;
+$nhsSuccess: #006400;
+$nhsError: #a7171a;
```

Comment:
Again, do we need styling for info and warning messages, even if they are not used right now?

Reply:
See above comment on your comment @ReubenRobertsHEE . Second option is my favourite.